Documentation/BuildInstructionsLadybird.md

```diff
@@ -52,7 +52,7 @@ nix-shell --command bash Ladybird
 
 On macOS:
 
-Note that Xcode 13.x does not have sufficient C++20 support to build ladybird. Xcode 14 versions before 14.3 might crash while building ladybird. Xcode 14.3 or clang from homebrew may be required to successfully build ladybird.
+Note that Xcode 13.x does not have sufficient C++23 support to build ladybird. Xcode 14 versions before 14.3 might crash while building ladybird. Xcode 14.3 or clang from homebrew may be required to successfully build ladybird.
```

Comment:
Are the remarks about Xcode versions correct for C++20 too?

Reply:
Well, the remark about "might crash" is true. So i'll just yeet the first sentence.